onboard-server/src/main/java/com/objectcomputing/checkins/services/onboard/onboardeeprofile/OnboardingProfileRepository.java

```diff
@@ -9,50 +10,73 @@
 
 import javax.validation.constraints.NotNull;
 import java.time.LocalDate;
-import java.util.Optional;
 import java.util.UUID;
 
-@JdbcRepository (dialect = Dialect.POSTGRES)
+@JdbcRepository(dialect = Dialect.POSTGRES)
 public interface OnboardingProfileRepository extends ReactorCrudRepository<OnboardingProfile, UUID> {
+        
+        @Override
+        @Query(value = "SELECT onboard_profile_id, " +
+                        "PGP_SYM_DECRYPT(cast(mp.firstName as bytea),'${aes.key}') as firstName, " +
+                        "PGP_SYM_DECRYPT(cast(mp.middleName as bytea),'${aes.key}') as middleName," +
+                        "PGP_SYM_DECRYPT(cast(mp.lastName as bytea),'${aes.key}') as lastName," +
+                        "PGP_SYM_DECRYPT(cast(mp.socialSecurityNumber as bytea),'${aes.key}') as socialSecurityNumber,"
+                        +
+                        "PGP_SYM_DECRYPT(cast(mp.birthDate as bytea),'${aes.key}') as birthDate," +
+                        "PGP_SYM_DECRYPT(cast(mp.currentAddress as bytea),'${aes.key}') as currentAddress," +
+                        "PGP_SYM_DECRYPT(cast(mp.previousAddress as bytea),'${aes.key}') as previousAddress," +
+                        "PGP_SYM_DECRYPT(cast(mp.phoneNumber as bytea),'${aes.key}') as phoneNumber," +
+                        "PGP_SYM_DECRYPT(cast(mp.secondPhoneNumber as bytea),'${aes.key}') as secondPhoneNumber," +
+                        "PGP_SYM_DECRYPT(cast(mp.personalEmail as bytea),'${aes.key}') as personalEmail " +
+                        "FROM \"onboard_profile\" mp " +
+                        "WHERE (onboard_profile_id = :onboardProfileId) ", nativeQuery = true)
```

Comment:
Same issue with the id. Do a search for 'onboard_profile_id' and replace with id in these files.